trieste/acquisition/function/active_learning.py

```diff
@@ -26,12 +26,13 @@
 
 from ...data import Dataset
```

Comment:
`IntegratedVarianceReduction` acq function here requires conditional predictions and there is a method check that can be eliminated, model instance can be checked instead